config-example

```diff
@@ -3,21 +3,17 @@
 # the URL to your server
 HOST="https://cloud.example.com"
 
-# enter the service that is providing the dav-service (eg. "owncloud"/"baikal")
+# enter the service that is providing the dav-service (eg. "owncloud"/"baikal"/"radicale")
 SERVICE="baikal"
 
 # enter your login-username
 DAVUSER="user"
```

Comment:
same applys to user
sorry for not telling you before

Reply:
No, we need this one to construct the URL. I admit it's duplication, but I guess the importance is to get the password out of the command line.